### DISCUSSION TITLE
Getting a configuration error on the Raspberry Pi when published with code trimming.

### DISCUSSION THREAD

**Discussion by rgelb** [quartznet/quartznet#2145] [General]
I have a simple .NET 7 console app that works fine on Windows.  I created a publish profile for my RPi and turned code trimming on.   The deployment mode is `Self-contained`.  

When deployed to the Raspberry Pi 4, it blows up on startup with `Unknown configuration property 'quartz.scheduler.jobFactory.type'`.   
However, when I redeployed with code trimming off, it worked fine.  This is fine, but without code trimming the deployment is 230 files vs 90.  

At some point something is ripped out of the deployed code that causes it to fail.

P.S.  I can no longer replicate the problem.

**Comment by lahma** [comment]
Would you be able to to decipher what is going on if someone would give you this vague description? A repro as public GitHub repository would be most welcome, that would show what kind of code you are using and even what version of Quartz you are using.
